new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporter.java

```diff
@@ -0,0 +1,1007 @@
+    deletePersistentVolumeClaim("pvc-prometheus",monitoringNS);
+    deletePersistentVolume("pv-testprometheus");
+    deletePersistentVolumeClaim("pvc-grafana",monitoringNS);
+    deletePersistentVolume("pv-testgrafana");
+    deleteNamespace(monitoringNS);
+    uninstallDeploymentService(webhookDepl, webhookService);
+    uninstallDeploymentService(coordinatorDepl, coordinatorService);
+    uninstallMonitoringExporter();
+
+    // uninstall NGINX release
+    if (nginxHelmParams != null) {
+      assertThat(uninstallNginx(nginxHelmParams))
+          .as("Test uninstallNginx returns true")
+          .withFailMessage("uninstallNginx() did not return true")
+          .isTrue();
+    }
+  }
+
+  /**
+   * Create a persistent volume and persistent volume claim.
+   * @param nameSuffix unique nameSuffix for pv and pvc to create
+   * @throws IOException when creating pv path fails
+   */
+  private void createPvAndPvc(String nameSuffix) throws IOException {
+    logger.info("creating persistent volume and persistent volume claim");
+    // create persistent volume and persistent volume claims
+    Path pvHostPath = assertDoesNotThrow(
+        () -> createDirectories(get(PV_ROOT, this.getClass().getSimpleName(), "monexp" + "-persistentVolume")),
+            "createDirectories failed with IOException");
+    logger.info("Creating PV directory {0}", pvHostPath);
+    assertDoesNotThrow(() -> deleteDirectory(pvHostPath.toFile()), "deleteDirectory failed with IOException");
+    assertDoesNotThrow(() -> createDirectories(pvHostPath), "createDirectories failed with IOException");
+
+    V1PersistentVolume v1pv = new V1PersistentVolume()
+        .spec(new V1PersistentVolumeSpec()
+            .addAccessModesItem("ReadWriteMany")
+            .storageClassName(nameSuffix)
+            .volumeMode("Filesystem")
+            .putCapacityItem("storage", Quantity.fromString("10Gi"))
+            .persistentVolumeReclaimPolicy("Retain")
+            .accessModes(Arrays.asList("ReadWriteMany"))
+            .hostPath(new V1HostPathVolumeSource()
+                .path(pvHostPath.toString())))
+        .metadata(new V1ObjectMeta()
+            .name("pv-test" + nameSuffix)
+            .namespace(monitoringNS)
+            .putLabelsItem("weblogic.domainUid", domain1Uid));
+
+
+    V1PersistentVolumeClaim v1pvc = new V1PersistentVolumeClaim()
+        .spec(new V1PersistentVolumeClaimSpec()
+            .addAccessModesItem("ReadWriteMany")
+            .storageClassName(nameSuffix)
+            .volumeName("pv-test" + nameSuffix)
+            .resources(new V1ResourceRequirements()
+                .putRequestsItem("storage", Quantity.fromString("10Gi"))))
+        .metadata(new V1ObjectMeta()
+            .name("pvc-" + nameSuffix)
+            .namespace(monitoringNS)
+            .putLabelsItem("weblogic.domainUid", domain1Uid));
+    createPVPVCAndVerify(v1pv,v1pvc, "weblogic.domainUid=" + domain1Uid, monitoringNS);
+  }
+
+  /**
```

Comment:
This can be moved to a utils class.

Reply:
I use util class but generated pv and pvc are custom objects for grafana, prom, they are different from domain pv/pvc, so I create these objects in the test class but use util method createPVPVCAndVerify from commontestutils to generate them, please let me know

Reply:
My comment was for method replaceStringInFile()

Reply:
sorry for misunderstanding, moved it to test utils